src/publicodes-state/hooks/useRule/useContent.ts

```diff
@@ -62,6 +62,17 @@ export default function useContent({ dottedName, rule }: Props) {
     return plancherValue ?? 0
   }, [rule])
 
+  const plafond = useMemo<number>(() => {
+    // By default, the plafond is 1 000 000
+    const plafondValue = rule?.rawNode['plafond']
+
+    // TODO: Deal with the case where the plafond needs to be evaluated.
+    if (typeof plafondValue === 'string') {
+      return 1000000
```

Comment:
Pour gagner en lisibilité

```suggestion
      return 1_000_000
```